x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/InternalRealmsTests.java

```diff
@@ -87,6 +88,15 @@ public void testLicenseLevels() {
         }
     }
 
+    public void testJwtRealmDependsOnBuildType() {
+        // Whether the JWT realm is registered depends on the build type
+        if (Build.CURRENT.isSnapshot()) {
+            assertThat(InternalRealms.isInternalRealm("jwt"), is(true));
+        } else {
+            assertThat(InternalRealms.isInternalRealm("jwt"), is(false));
```

Comment:
Nit. Is it possible to use JwtRealmSettings.TYPE instead of "jwt"? I think that would be consistent with other uses of realm type constants in this class, such as NativeRealmSettings.TYPE, LdapRealmSettings.LDAP_TYPE, and LdapRealmSettings.AD_TYPE.